quizara-backend/models/leaderboard.model.js

```diff
@@ -0,0 +1,16 @@
+const mongoose  = require("mongoose")
+const  {Schema} = mongoose
+
+const LeaderboardSchema = new mongoose.Schema({
```

Comment:
You have already destructured and took out Schema on line 2, so here you don't need to repeat again, simply 
do `new Schema`